downstream/assemblies/platform/assembly-network-ports-protocols.adoc

```diff
@@ -263,23 +267,23 @@ a|Open *only* if the internal database is used. Otherwise, this port should not
 |===
 |URL |Required for
 |link:https://registry.redhat.io:443[https://registry.redhat.io:443] |Access to container images provided by Red Hat and partners
-| `cdn.quay.io:443` | Access to container images provided by Red Hat and partners
-| `cdn01.quay.io:443` | Access to container images provided by Red Hat and partners
-| `cdn02.quay.io:443` | Access to container images provided by Red Hat and partners
-| `cdn03.quay.io:443` | Access to container images provided by Red Hat and partners
+|`cdn.quay.io:443` | Access to container images provided by Red Hat and partners
+|`cdn01.quay.io:443` | Access to container images provided by Red Hat and partners
+|`cdn02.quay.io:443` | Access to container images provided by Red Hat and partners
+|`cdn03.quay.io:443` | Access to container images provided by Red Hat and partners
 |===
 
 [IMPORTANT]
 ====
-Image manifests and filesystem blobs are served directly from `registry.redhat.io`. 
-However, from 1st of May 2023, filesystem blobs are served from `quay.io` instead. 
+Image manifests and filesystem blobs are served directly from `registry.redhat.io`.
+However, from 1st of May 2023, filesystem blobs are served from `quay.io` instead.
```

Comment:
Per IBM style guide, do not use ordinal numerals (that is, numerals that use the abbreviations st, nd, rd, and th) to specify a date. See 'General guidance for dates' section in the IBM Style Guide (page 103). 
Suggest revising 1st of May 2023 -> 1 May 2023.